### PR TITLE
[EN] Difficulties/osu! (index) and Difficulties/osu!taiko (index)

### PR DESCRIPTION
**Pull request by MegaApplePi**

### CODE REVIEW

File: wiki/Difficulties/osu!/en.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# osu!
+
+These are the five difficulties a [beatmapper](/wiki/beatmapper) can [map](/wiki/map) for an osu!standard [mapset](/wiki/mapset).
+The articles below explains how to setup your beatmap for a specific difficulty and what you should be aware of when making them.
```
Comment (oralekin) [inline wiki/Difficulties/osu!/en.md]: `set your beatmap up`? Native pls fix

Comment (MegaApplePi) [inline wiki/Difficulties/osu!/en.md]: that sounds ok, will change to both

Comment (Shadowtail117) [inline wiki/Difficulties/osu!/en.md]: "setup" is a noun, "set up" is a verb. It should be "how to set up your beatmap".

File: wiki/Difficulties/osu!/en.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# osu!
+
+These are the five difficulties a [beatmapper](/wiki/beatmapper) can [map](/wiki/map) for an osu!standard [mapset](/wiki/mapset).
```
Comment (TPGPL) [inline wiki/Difficulties/osu!/en.md]: There are

File: wiki/Difficulties/osu!taiko/en.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# osu!taiko
+
+These are the five difficulties a [beatmapper](/wiki/beatmapper) can [map](/wiki/map) for an osu!taiko [mapset](/wiki/mapset).
```
Comment (TPGPL) [inline wiki/Difficulties/osu!taiko/en.md]: There are

File: wiki/Difficulties/osu!/en.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+The articles below explains how to set your beatmap up for a specific difficulty and what you should be aware of when making them.
```
Comment (TPGPL) [inline wiki/Difficulties/osu!/en.md]: set up your beatmap?

File: wiki/Difficulties/osu!taiko/en.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+The articles below explains how to set your beatmap up for a specific difficulty and what you should be aware of when making them.
```
Comment (TPGPL) [inline wiki/Difficulties/osu!taiko/en.md]: set up your beatmap

File: wiki/Difficulties/osu!/en.md [hunk 6]
```diff
@@ -0,0 +1,10 @@
+# osu!
+
+There are the five difficulties a [beatmapper](/wiki/beatmapper) can [map](/wiki/map) for an osu!standard [mapset](/wiki/mapset).
```
Comment (RaikireHiuduo) [inline wiki/Difficulties/osu!/en.md]: "There are the five..." sounds odd to me.

`There are five difficulties a [beatmapper](/wiki/beatmapper) can [map](/wiki/map) for an osu!standard [mapset](/wiki/mapset).`

File: wiki/Difficulties/osu!/en.md [hunk 7]
```diff
@@ -0,0 +1,10 @@
...
+The articles below explains how to set your beatmap up for a specific difficulty and what you should be aware of when making them.
```
Comment (RaikireHiuduo) [inline wiki/Difficulties/osu!/en.md]: Removing `you` dependency and some rewriting.

``The articles below explains how to prepare a beatmap for a specific difficulty and what to be aware of in the process of making the beatmap.``

File: wiki/Difficulties/osu!taiko/en.md [hunk 8]
```diff
@@ -0,0 +1,10 @@
+# osu!taiko
+
+There are the five difficulties a [beatmapper](/wiki/beatmapper) can [map](/wiki/map) for an osu!taiko [mapset](/wiki/mapset).
```
Comment (RaikireHiuduo) [inline wiki/Difficulties/osu!taiko/en.md]: ditto, dood.

`There are five difficulties a [beatmapper](/wiki/beatmapper) can [map](/wiki/map) for an osu!taiko [mapset](/wiki/mapset).`

File: wiki/Difficulties/osu!taiko/en.md [hunk 9]
```diff
@@ -0,0 +1,10 @@
...
+The articles below explains how to set your beatmap up for a specific difficulty and what you should be aware of when making them.
```
Comment (RaikireHiuduo) [inline wiki/Difficulties/osu!taiko/en.md]: ditto, dood.
``The articles below explains how to prepare a beatmap for a specific difficulty and what to be aware of in the process of making the beatmap.``

File: wiki/Difficulties/osu!taiko/en.md [hunk 10]
```diff
@@ -0,0 +1,10 @@
...
+
+- ![Easy icon](/wiki/shared/Easy-t.png) [Futsuu](/wiki/Futsuu)
+- ![Normal icon](/wiki/shared/Normal-t.png) [Kantan](/wiki/Kantan)
+- ![Hard icon](/wiki/shared/Hard-t.png) [Muzukashii](/wiki/Muzukashii)
+- ![Insane icon](/wiki/shared/Insane-t.png) [Oni](/wiki/Oni)
+- ![Expert icon](/wiki/shared/Expert-t.png) [Ura Oni](/wiki/Ura_Oni) (also known as _Inner Oni_)
```
Comment (RaikireHiuduo) [inline wiki/Difficulties/osu!taiko/en.md]: Whole list. Futsuu is **Normal** difficulty. Kantan is _Easy_.

```Markdown
- ![Easy icon](/wiki/shared/Easy-t.png) [Kantan](/wiki/Kantan) (also known as _Easy_)
- ![Normal icon](/wiki/shared/Normal-t.png) [Futsuu](/wiki/Futsuu) (also known as _Normal_)
- ![Hard icon](/wiki/shared/Hard-t.png) [Muzukashii](/wiki/Muzukashii) (also known as _Hard_)
- ![Insane icon](/wiki/shared/Insane-t.png) [Oni](/wiki/Oni)
- ![Expert icon](/wiki/shared/Expert-t.png) [Ura Oni](/wiki/Ura_Oni) (also known as _Inner Oni_)
```

File: wiki/Difficulties/osu!/en.md (hunk 6 above)
Comment (kj415j45) [inline wiki/Difficulties/osu!/en.md]: > five **basic** difficulties

Comment (MegaApplePi) [inline wiki/Difficulties/osu!/en.md]: but there are only 5 [categories]: Easy, Normal, Hard, Insane, and Expert; as of now

File: wiki/Difficulties/osu!taiko/en.md (hunk 10 above)
Comment (MegaApplePi) [inline wiki/Difficulties/osu!taiko/en.md]: I would guess that the icon should do the job

**Pull request merged**